src/main/scala/org/globalforestwatch/summarystats/carbon_sensitivity/CarbonSensitivityGridSources.scala

```diff
@@ -37,13 +37,13 @@ case class CarbonSensitivityGridSources(gridTile: GridTile,
   val treeCoverGain: TreeCoverGain = TreeCoverGain(gridTile, kwargs)
   val mangroveBiomassExtent: MangroveBiomassExtent = MangroveBiomassExtent(gridTile, kwargs)
   val treeCoverLossDrivers: TreeCoverLossDrivers = TreeCoverLossDrivers(gridTile, kwargs)
-  val ecozones: Ecozones = Ecozones(gridTile, kwargs)
+  val ecozones: FaoEcozones2000 = FaoEcozones2000(gridTile, kwargs = kwargs)
```

Comment:
Remove 'kwargs =' to be consistent with the calls in all the adjacent fields.

Reply:
Done.

Reply:
It actually needs that `kwargs = kwargs` because the FAO ecozones layer can handle different variants. It was a feature I added a while ago for getting zonal stats from different versions of my model.